config/locales/simple_form.fa.yml

```diff
@@ -5,13 +5,19 @@ fa:
       defaults:
         avatar: یکی از قالب‌های PNG یا  GIF یا JPG. بیشترین اندازه ۲ مگابایت. تصویر به اندازهٔ ۱۲۰×۱۲۰ پیکسل تبدیل خواهد شد.
         display_name: '%{count} حرف باقی مانده'
+            one: '<span class="name-counter">1</span> حرف باقی مانده'
```

Comment:
I fixed the above. But I don't see from the logs what is still wrong. Could you please point me to any remaining failure?